cmd/build-push-image/main.go

```diff
@@ -194,13 +200,63 @@ func main() {
 	}
 }
 
+func addNexusBuildArgs(args []string, opts options) (ret_args []string) {
+	// log.Printf("nexusURL: %s", opts.nexusURL)
+	if strings.TrimSpace(opts.nexusURL) != "" {
+		nexusUrl, err := url.Parse(opts.nexusURL)
+		if err != nil {
+			log.Printf("Skip adding nexus build args: could not parse nexus url (%s): %s", opts.nexusURL, err)
+			return args
+		}
+		if nexusUrl.Host == "" {
+			log.Printf("Skip adding nexus build args: could not get host in nexus url (%s)", opts.nexusURL)
```

Comment:
Same here, I think we might also return an error here ...

Reply:
Changed with new commit.